packages/config/src/android/Updates.ts

```diff
@@ -9,8 +22,12 @@ export function getUpdateUrl(config: ExpoConfig, username: string | null) {
   return `https://exp.host/@${user}/${config.slug}`;
 }
 
-export function getSDKVersion(config: ExpoConfig) {
-  return typeof config.sdkVersion === 'string' ? config.sdkVersion : null;
+export function getRuntimeVersion(config: ExpoConfig) {
+  return typeof config.runtimeVersion === 'string' ? config.runtimeVersion : null;
+}
+
+export function getSDKVersion(config: ExpoConfig, sdkVersion: string | null) {
+  return typeof config.sdkVersion === 'string' ? config.sdkVersion : sdkVersion;
```

Comment:
if `config.sdkVersion` is missing then there is no sdkVersion defined, if in some cases that change is necessary this is not a place to handle it